common/test/acceptance/pages/lms/instructor_dashboard.py

```diff
@@ -678,72 +714,69 @@ def discussion_topics_visible(self):
         return (self.q(css=self._bounded_selector('.cohort-course-wide-discussions-nav')).visible and
                 self.q(css=self._bounded_selector('.cohort-inline-discussions-nav')).visible)
 
-    def select_discussion_topic(self, key):
+    def divided_discussion_heading_is_visible(self, key):
         """
-        Selects discussion topic checkbox by clicking on it.
+        Returns the visibility of discussion topic headings.
         """
-        self.q(css=self._bounded_selector(".check-discussion-subcategory-%s" % key)).first.click()
+        form_heading_css = '%s %s' % (self.discussion_form_selectors[key], '.subsection-title')
+        discussion_heading = self.q(css=self._bounded_selector(form_heading_css))
+
+        if len(discussion_heading) == 0:
+            return False
+        return discussion_heading.first.text[0]
```

Comment:
This is returning text instead of a boolean, correct? Just trying to understand the method (I know it was pre-existing).